lighthouse-core/lib/traces/tracing-processor.js

```diff
@@ -134,12 +129,16 @@ class TraceProcessor {
    * @return {!Array<{percentile: number, time: number}>}
    */
   static getRiskToResponsiveness(tabTrace, startTime, endTime, percentiles) {
-    const events = tabTrace.processEvents.filter(e => e.ts !== 0);
-    const lastEvent = events[events.length - 1];
+    const navStart = tabTrace.navigationStartEvt.ts;
+    const traceStart = tabTrace.processEvents
```

Comment:
should we move these two (traceStart/traceEnd) into tabTrace ?  i'm okay either way.